lmnet/executor/evaluate.py

```diff
@@ -36,15 +55,14 @@ def evaluate(config, restore_path):
 
     DatasetClass = config.DATASET_CLASS
     ModelClass = config.NETWORK_CLASS
-    network_kwargs = dict((key.lower(), val) for key, val in config.NETWORK.items())
-    dataset_kwargs = dict((key.lower(), val) for key, val in config.DATASET.items())
+    network_kwargs = {key.lower(): val for key, val in config.NETWORK.items()}
 
     if "test" in DatasetClass.available_subsets:
         subset = "test"
     else:
         subset = "validation"
 
-    validation_dataset = DatasetIterator(DatasetClass(subset=subset, **dataset_kwargs), seed=0)
+    validation_dataset = setup_dataset(config, subset, rank=0)
```

Comment:
Why do you change parameter `seed` to `rank` ? 
I think we maybe should keep `seed` param, because `seed=0` means that we can get the same result in every evaluation  time.

Reply:
I used the same argument names as `setup_dataset` in `train.py` but as you mentioned, `seed` is appropriate in this case. Thank you.